### PR TITLE
Getting Started with Directus and Flutter

### PR DESCRIPTION
**Pull request by Claradev32**
First draft

### CODE REVIEW

File: crud-with-flutter-and-directus/index.md [hunk 1]
```diff
@@ -0,0 +1,561 @@
...
+    
+![Navigating to the blog single page](./Screenshot%202024-04-17%20at%2018.27.22.png)
+
+## Summary
+Throughout this tutorial, you've learned how to build a CRUD application using Directus and Flutter. We started by creating a new project, set up environment variables and everything we need to call Directus. Then we went further to create pages, and posts collections in Directus and integrate the Flutter application with Directus by making API requests to perform CRUD operations.
+
+Now that you've acquired this knowledge, what would you like to build next with the Directus CMS? Perhaps you can improve on this project by adding authentication to protect the application, and form validation/serialization. To learn more about Dirctus and other interesting features it provides, visit the [official documentation](https://docs.directus.io/).
```
Comment (phazonoverload) [inline crud-with-flutter-and-directus/index.md:561]:
```suggestion
Throughout this tutorial, you've learned how to build a Flutter application that uses data from a Directus project. We started by creating a new project, set up environment variables and everything we need to call Directus. We then created pages and posts collections in Directus and integrated them with the the Flutter.
```

**Review by phazonoverload — CHANGES_REQUESTED** [review]
Good post. Some comments that need addressing. There is still some work needed to bring it to the standard format. Please also use the consistent `directory` term throughout.

**Comment by Claradev32** [comment]
Hi @phazonoverload,

I have worked on your reviews.

Thank you.

**Comment by phazonoverload** [comment]
All of my review comments were not incorporated. Please can you make sure they are accurately and then request a re-review.

File: crud-with-flutter-and-directus/index.md [hunk 2]
```diff
@@ -0,0 +1,561 @@
...
+- **title**: Text input field
+- **content**: WYSIWYG input field
+- **status**: Dropdown filed
+- **date_created**: Datetime field
+    
+![Creating the posts collection in Directus CMS](./Screenshot%202024-04-17%20at%2011.13.45.png)
+
+Now add a couple of blog data to the `posts` collection.
+    
+![Adding entries to Directus collection](./Screenshot%202024-04-17%20at%2011.26.32.png)
```
[the remark this answers is not in the document]
Comment (Claradev32) [reply crud-with-flutter-and-directus/index.md]: Done

**Comment by Claradev32** [comment]
@phazonoverload I've addressed the feedback and made the requested changes. Could you please take another look?

**Comment by phazonoverload** [comment]
Thank you for writing this, and we are happy to consider it done in it's current state. My team will schedule it in to be published. 

This repo is just for authoring/reviewing - the actual piece will be published via a Directus project, so we'll take it from here. I'll close this issue to keep the main branch clean, but this is a acceptance of your work.

Could you email me at [devrel@directus.io](mailto:devrel@directus.io) and we'll share what we need to get you paid. Thanks!

**Pull request closed without merging**